src/umpire/strategy/DynamicPoolList.hpp

```diff
@@ -34,8 +34,17 @@ namespace strategy {
  */
 class DynamicPoolList : public AllocationStrategy {
  public:
+  /*!
+   * \brief Coalescing Heuristic functions for Percent-Releasable and Blocks-Releasable. Both have
+   * the option to reallocate to High Watermark instead of actual size of the pool (actual size is
+   * currently the default).
+   */
   static PoolCoalesceHeuristic<DynamicPoolList> percent_releasable(int percentage);
+  static PoolCoalesceHeuristic<DynamicPoolList> percent_releasable_hwm(int percentage);
```

Comment:
Should we combine these two functions by using default parameters?

```suggestion
  static PoolCoalesceHeuristic<DynamicPoolList> percent_releasable(int percentage, bool resize_to_hwm = false);
```

Reply:
See https://rzlc.llnl.gov/jira/browse/UM-1022